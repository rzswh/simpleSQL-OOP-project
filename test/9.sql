CREATE DATABASE OOP;
USE OOP;
CREATE TABLE poi(lkj INT, mnb INT, bvc INT, dfj CHAR, PRIMARY KEY(mnb));
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (1, 2, 3, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (2, 3, 4, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (3, 4, 5, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (4, 5, 6, "a");
INSERT INTO poi(lkj, mnb, bvc, dfj) VALUES (5, 6, 7, "a");
INSERT INTO poi(mnb, bvc, dfj) VALUES (7, 7, "a");
DELETE FROM poi  WHERE lkj>3 OR mnb=7 AND lkj=10;
SELECT * from poi;
DROP DATABASE OOP;
