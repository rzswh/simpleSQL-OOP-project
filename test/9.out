lkj	mnb	bvc	dfj
1	2	3	a
2	3	4	a
3	4	5	a
NULL	7	7	a
